### PR TITLE
harsilspatel/templating engine

### PR DESCRIPTION
**Pull request by harsilspatel**

### CODE REVIEW

**Comment by harsilspatel** [comment]
@khanguslee the only drawback of refactoring all the code of sidebar into one single file is that, it will not highlight the current page we're on in the sidebar. not sure how to go about fixing that...

**Comment by harsilspatel** [comment]
taken care of highlighting :)

**Comment by khanguslee** [comment]
Look how clean it looks now. Will have a look after work. Good job @harsilspatel ! Appreciate it.

File: server/server.js [hunk 1]
```diff
@@ -36,13 +36,38 @@ sockets.init(server);
 server.listen(PORT, () => {
   console.log('Example app listening at', server.address().port);
 });
+const HEADER = 'MHP DAShboard';
+const sidebar = [
+  { file: 'index', title: 'Dashboard' },
+  { file: 'files', title: 'Download Files' },
```
Comment (khanguslee) [inline server/server.js]: Let's change this to be `{ file: 'download-files', title: 'Download Files' },` instead so we don't get a clash in endpoints?

File: server/js/sockets.js [hunk 2]
```diff
@@ -3,7 +3,7 @@
  */
 const sockets = {};
 const mqtt = require('mqtt');
-const io = require('socket.io');
+// const io = require('socket.io');
```
Comment (khanguslee) [inline server/js/sockets.js]: Remove this comment.

**Review by khanguslee — CHANGES_REQUESTED** [review]
Nearly there!

File: server/server.js (hunk 1 above)
Comment (harsilspatel) [reply server/server.js]: ohhh i get it now! alright i'll do that :)

**Review by khanguslee — APPROVED** [review]
Workin' good. You don't know how useful this will be in the long run ;)

**Pull request merged**